automod/triggers.go

```diff
@@ -47,6 +48,12 @@ func (r BaseRegexTrigger) UserSettings() []*SettingDef {
 			Min:  1,
 			Max:  250,
 		},
+		{
+			Name:    "FixText the message content and check the result aswell",
```

Comment:
```suggestion
			Name:    "Also match Zalgo and confusable characters",
```
For all instances where you have added this.

Reply:
I forgot to change those oops